### DISCUSSION TITLE
server to server subtree sync

### DISCUSSION THREAD

**Discussion by Arbel-arad** [TriliumNext/Trilium#4740] [General]
i have a trilium server which has both personal data and documentation that i would like to be public.
making the server externally available isn't great for security, so i would like to avoid that (so the share feature won't work)

is it possible to set up another server, that is externally accessible and only sync a specific subtree to it from my current server?

up to this point my solution has been to manually export the notes to html and put them on a static webpage which is always outdated.

**Comment by meichthys** [comment]
The recommended solution here is to use a reverse proxy in front of the server to improve security.
What you are looking for does not exist, but could probably be scripted with the etapi
